wandb/internal/update.py

```diff
@@ -20,7 +20,18 @@ def _find_available(current_version):
 
     # Return if no update is available
     pip_prerelease = False
+    yanked = False
+    deleted = False
     parsed_current_version = parse_version(current_version)
+
+    # Check if current version has been yanked or deleted
+    if current_version in release_list:
+        for item in data["releases"][current_version]:
+            yanked = yanked or item["yanked"]
```

Comment:
There is a yank message string that you get back from pypi on yank.  It would be nice to capture that yank message